localstack/services/secretsmanager/provider.py

```diff
@@ -133,7 +133,12 @@ def cancel_rotate_secret(
     def create_secret(
         self, context: RequestContext, request: CreateSecretRequest
     ) -> CreateSecretResponse:
+        if "ClientRequestToken" not in request:
```

Comment:
There is some duplication about this check, I think it'd be nicer to have a similar pattern to `_raise_if_invalid_secret_id`.

Reply:
Done! ✅